collect_app/src/main/java/org/odk/collect/android/activities/FirstLaunchActivity.kt

```diff
@@ -117,7 +117,7 @@ class FirstLaunchActivity : LocalizedActivity(), CollectComposeThemeProvider {
                 text = SpannableStringBuilder()
                     .append(getString(org.odk.collect.strings.R.string.dont_have_project))
                     .append(" ")
-                    .color(getThemeAttributeValue(context, com.google.android.material.R.attr.colorAccent)) {
+                    .color(getThemeAttributeValue(context, androidx.appcompat.R.attr.colorAccent)) {
```

Comment:
We should be using a [Material color](https://github.com/material-components/material-components-android/blob/master/docs/theming/Color.md) here instead of an AppCompat one I think. Same goes for the other places we're using `colorAccent`.